src/Groups/GAPGroups.jl

```diff
@@ -710,7 +710,8 @@ end
     is_conjugate(G::GAPGroup, x::GAPGroupElem, y::GAPGroupElem)
 
 Return whether `x` and `y` are conjugate elements in `G`,
-i.e., there is an element $z$ in `G` such that `x^`$z$ equals `y`.
+i.e., there is an element `z` in `G` such that `x^z` equals `y`.
```

Comment:
Here you replaced  `` `x^`$z$ `` by `` `x^z` `` (I think @ThomasBreuer used the former deliberately, though one may argue about that).

Below you introduced a new occurrence of `` `H^`$z$ `` in the `is_conjugate(G::GAPGroup, H::GAPGroup, K::GAPGroup)` docstring.

Finally in the `is_conjugate_with_data(G::Group, H::Group, K::Group)` docstring very similar new text was added by here with `` `H^z = K` ``

I have no strong opinion which to use, as long as their is some consistency or rule

Reply:
Yeah, I guess I felt like it looked strange when the the z needed to appear as a literal (ex "returns `(true, z)`") to have it typeset in LaTeX in another part of the docstring; I'll make these consistent (no LaTeX).